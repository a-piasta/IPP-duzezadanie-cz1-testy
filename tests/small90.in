newRoute;333;hxljfzxuspbzzedftidowqtqoezci;cbwjgkdkgmbutithtasjbkwxmqx
repairRoad;hxljfzxuspbzzedftidowqtqoezci;fehdeenfysolfsylimcv;5
newRoute;21;dbhhffqlpvxxkbrdjlm;nmsvniihhvasfxvkgcitbwpxme
newRoute;847;shvytjjyvialigb;nszntlhuafjfwnzouuopdxna
getRouteDescription;333
getRouteDescription;333
extendRoute;847;gkmjfjzzqygpffrhazmsbpynfod
extendRoute;333;dbhhffqlpvxxkbrdjlm
getRouteDescription;21
addRoad;hxljfzxuspbzzedftidowqtqoezci;shvytjjyvialigb;26;1
repairRoad;mudwjpofexnlglozqlwcrylnc;neooqkldckcfolpt;-7
removeRoad;shvytjjyvialigb;hxljfzxuspbzzedftidowqtqoezci
removeRoad;hxljfzxuspbzzedftidowqtqoezci;shvytjjyvialigb
removeRoad;shvytjjyvialigb;hxljfzxuspbzzedftidowqtqoezci
addRoad;nszntlhuafjfwnzouuopdxna;fehdeenfysolfsylimcv;19;-13
removeRoad;fehdeenfysolfsylimcv;nszntlhuafjfwnzouuopdxna
newRoute;168;nmsvniihhvasfxvkgcitbwpxme;dbhhffqlpvxxkbrdjlm
repairRoad;gkmjfjzzqygpffrhazmsbpynfod;mudwjpofexnlglozqlwcrylnc;9
removeRoad;shvytjjyvialigb;hxljfzxuspbzzedftidowqtqoezci
extendRoute;168;gkmjfjzzqygpffrhazmsbpynfod
addRoad;gkmjfjzzqygpffrhazmsbpynfod;neooqkldckcfolpt;27;-2
repairRoad;fehdeenfysolfsylimcv;nszntlhuafjfwnzouuopdxna;-14
newRoute;772;nszntlhuafjfwnzouuopdxna;neooqkldckcfolpt
removeRoad;fehdeenfysolfsylimcv;nszntlhuafjfwnzouuopdxna
getRouteDescription;333
addRoad;nszntlhuafjfwnzouuopdxna;dbhhffqlpvxxkbrdjlm;0;-10
newRoute;884;shvytjjyvialigb;dbhhffqlpvxxkbrdjlm
getRouteDescription;884
extendRoute;21;nszntlhuafjfwnzouuopdxna
extendRoute;21;mudwjpofexnlglozqlwcrylnc
newRoute;651;nmsvniihhvasfxvkgcitbwpxme;nmsvniihhvasfxvkgcitbwpxme
extendRoute;651;nmsvniihhvasfxvkgcitbwpxme
addRoad;dbhhffqlpvxxkbrdjlm;gkmjfjzzqygpffrhazmsbpynfod;0;-1
addRoad;nszntlhuafjfwnzouuopdxna;shvytjjyvialigb;14;9
newRoute;127;mudwjpofexnlglozqlwcrylnc;hxljfzxuspbzzedftidowqtqoezci
getRouteDescription;847
repairRoad;hxljfzxuspbzzedftidowqtqoezci;shvytjjyvialigb;-7
addRoad;nszntlhuafjfwnzouuopdxna;shvytjjyvialigb;17;14
repairRoad;mudwjpofexnlglozqlwcrylnc;shvytjjyvialigb;2
extendRoute;21;cbwjgkdkgmbutithtasjbkwxmqx
addRoad;fehdeenfysolfsylimcv;hxljfzxuspbzzedftidowqtqoezci;10;2
extendRoute;50;dbhhffqlpvxxkbrdjlm
extendRoute;847;neooqkldckcfolpt
getRouteDescription;651
repairRoad;nszntlhuafjfwnzouuopdxna;hxljfzxuspbzzedftidowqtqoezci;-12
addRoad;shvytjjyvialigb;nmsvniihhvasfxvkgcitbwpxme;12;3
extendRoute;50;nmsvniihhvasfxvkgcitbwpxme
addRoad;dbhhffqlpvxxkbrdjlm;nszntlhuafjfwnzouuopdxna;21;-12
removeRoad;nszntlhuafjfwnzouuopdxna;shvytjjyvialigb
repairRoad;dbhhffqlpvxxkbrdjlm;mudwjpofexnlglozqlwcrylnc;-15
repairRoad;nmsvniihhvasfxvkgcitbwpxme;mudwjpofexnlglozqlwcrylnc;-9
repairRoad;cbwjgkdkgmbutithtasjbkwxmqx;fehdeenfysolfsylimcv;8
addRoad;nmsvniihhvasfxvkgcitbwpxme;cbwjgkdkgmbutithtasjbkwxmqx;24;-15
removeRoad;neooqkldckcfolpt;gkmjfjzzqygpffrhazmsbpynfod
extendRoute;127;mudwjpofexnlglozqlwcrylnc
getRouteDescription;847
repairRoad;neooqkldckcfolpt;nmsvniihhvasfxvkgcitbwpxme;-15
