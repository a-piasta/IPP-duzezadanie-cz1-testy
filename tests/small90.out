1: NIE
2: NIE
3: NIE
4: NIE
5: 
6: 
7: NIE
8: NIE
9: 
10: TAK
11: NIE
12: TAK
13: NIE
14: NIE
15: TAK
16: TAK
17: NIE
18: NIE
19: NIE
20: NIE
21: TAK
22: NIE
23: NIE
24: NIE
25: 
26: NIE
27: NIE
28: 
29: NIE
30: NIE
31: NIE
32: NIE
33: NIE
34: TAK
35: NIE
36: 
37: NIE
38: NIE
39: NIE
40: NIE
41: TAK
42: NIE
43: NIE
44: 
45: NIE
46: TAK
47: NIE
48: TAK
49: TAK
50: NIE
51: NIE
52: NIE
53: TAK
54: TAK
55: NIE
56: 
57: NIE
